ci-operator/step-registry/ipi/conf/azurestack/additional-ca/ipi-conf-azurestack-additional-ca-ref.metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"path": "ipi/conf/azurestack/additional-ca/ipi-conf-azurestack-additional-ca-ref.yaml",
```

Comment:
How about reanme the path to `ipi/conf/additional-ca/azurestack`, maybe in the future, we can re-use the folder for other cloudproviders.